tests/e2e/specs/orderList_spec.js

```diff
@@ -39,13 +39,14 @@ describe('my orders', () => {
 
   beforeEach(() => {
     cy.visit('/');
+    cy.createCustomer(customer);
     cy.login(customer);
   });
 
   it('shows my orders', () => {
     cy.createOrder(cartDraft1, orderDraft1);
     cy.createOrder(cartDraft2, orderDraft2);
-    cy.get('[data-test=my-orders-button]', { timeout: 20000 }).click();
+    cy.get('[data-test=my-orders-button]', { timeout: Cypress.config('graphqlTimeout') }).click();
```

Comment:
I would remove this one. Not because it's not necessary (as it's coming from a "login"), but because the login is so far away that is very unlikely that is still waiting.